test/quantization/test_quantize_fx.py

```diff
@@ -406,6 +406,122 @@ def forward(self, x):
         FileCheck().check_count('FakeQuantize = prim::GetAttr[name="activation_post_process', 4, exactly=True) \
                    .run(scripted.graph)
 
+    @skipIfNoFBGEMM
+    def test_custom_module_class(self):
+        class CustomModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(1, 1, 1)
+
+            def forward(self, x):
+                return self.conv(x)
+
+        class ObservedCustomModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, x):
+                pass
+
+            @classmethod
+            def from_float(cls, float_module):
```

Comment:
Where are the observers inserted for weights and activations?

Reply:
here we reusing the eager mode conv module, so there is no need to insert weight observers. the observer for activation is inserted outside of the custom module.

Reply:
How would this flow work for QAT? In that case weight observers (fake quant) needs to be inserted at prepare.

Reply:
it will use the qconfig from float_module to instantiate a weight observer and call it in the forward.

Reply:
the flow will be exactly the same

Reply:
Is it correct to say that in general it is the responsibility of the customModule owner to write a Observedclass that contains all the weight +activations observers except for the outputs?

Reply:
yes, that is correct